pnpm-lock.yaml

```diff
@@ -273,7 +273,222 @@ importers:
         specifier: ^5.8.0
         version: 5.10.0
 
-  tests/fixtures/root-config-test:
+  tests/fixtures/react-app-js-webpack:
```

Comment:
I'm averse to merging this because of lots of changes to the pnpm-lock.yaml. The CONTRIBUTING.md outlines how to avoid unnecessary changes